components/navbar/sidebar.snip.html

```diff
@@ -25,7 +25,7 @@
     <ul class="list-reset m0 p0 ampstart-label">
       {{#nav}}
         {{#link}}
-          <li class="ampstart-nav-item"><a class="ampstart-nav-link" href="{{{url}}}">{{text}}</a></li>
+          <li class="ampstart-nav-item"><a class="ampstart-nav-link" href="{{{url}}}">{{{text}}}</a></li>
```

Comment:
This kind of effect is possible with `letter-spacing`, but we decided not to go with this approach for 2 reasons:

1. The navbar is width is set to a `vw` unit and so the width can be variable depending on resolution. We'd have to add specific breakpoints change the letter spacing at the right moment. Using `space-between` keeps the letters spaced out no matter what the width is.
2. This method keeps the CSS separate from the content. For example if a developer needed to change the nav buttons, they would also need to fiddle with the letter spacing at each resolution to get the same effect.

I understand that this method is not ideal and if you'd rather not implement this, we could have a chat with the designer to remove this effect.